.github/workflows/build.yml

```
@@ -35,54 +35,98 @@ jobs:
-      - name: Run unit tests and generate coverage
-        run: dotnet-coverage collect "dotnet test VictoryCenter/VictoryCenter.UnitTests/VictoryCenter.UnitTests.csproj --configuration Release" -f xml -o "coverage.xml"
+      - name: Set up Docker Compose
+        run: docker compose -f docker-compose.yml up -d
 
-      - name: Run integration tests and merge coverage
-        run: dotnet-coverage merge "coverage.xml" "dotnet-coverage collect \"dotnet test VictoryCenter/VictoryCenter.IntegrationTests/VictoryCenter.IntegrationTests.csproj --configuration Release\" -f xml -o \"coverage2.xml\"" -o "coverage.xml"
+      - name: Wait for SQL Server to be ready
+        run: |
+          echo "Waiting for SQL Server to start..."
+          sleep 20
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)